### ISSUE TITLE
Structural abnormality of the chest wall

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5901]
**Preferred term label:** Structural abnormality of the chest wall 

**Synonyms**

**Definition (free text, please give PubMed ID)** Abnormal development of the thoracic skeleton resulting in an abnormal shape of the chest wall.

**Parent term (use hpo.jax.org/app)** Abnormality of the thorax HP:0000765

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for: Abnormality of the sternum HP:0000766; Abnormality of the rib cage HP:0001547

**Comment by pnrobinson** [comment]
I think this is the same as Abnormality of the thorax HP:0000765
All of the terms are structural. I am changing the label to Abnormal thorax morphology and will add this as a synoynm. I will revise the hierarchy as suggested.
